docs/product/sales.rst

```diff
@@ -0,0 +1,80 @@
+Sales and Marketing
```

Comment:
The sales pages was merged...but odd enough I don't see it online on geck.  It should be in the Product section as well.

Reply:
Nevermind..